webapp/src/main/java/org/apache/atlas/web/rest/BusinessPolicyREST.java

```diff
@@ -57,7 +57,7 @@ public void linkBusinessPolicy(@PathParam("policyId") final String policyGuid, f
         // Set request context parameters
         RequestContext.get().setIncludeClassifications(false);
         RequestContext.get().setIncludeMeanings(false);
-        RequestContext.get().getRequestContextHeaders().put("route", "business-policy-rest");
+        RequestContext.get().getRequestContextHeaders().put("x-atlan-route", "business-policy-rest");
```

Comment:
Overall requirement is still not clear to me, even if any client calling this API, it will be still writing the header then why you even need to add it?

I only see one use case where this API is called from different clients & also then clients must add the header & not the API itself

Also, imagine in the future we make this API generic (most likely to happen with Product<>Asset, Domain<>Asset linking) then this will not work anyway

Overall I feel clients should be responsible to set header & not the final API

Reply:
This is a specific implementation for the policy manager I don't see it getting generalised for any other user case - while the other use case can use the idea of this endpoint I prefer to keep them isolated from each other (while the Service layer can have some common code but the entry should be separate as these are particular API for a particular use case)

This is a specific header we are producing from the API code itself intentionally - to use them in the numaflow pipeline which is the extension of this API in the overall solution of policy-manger

The client will set it headers but we require specifics to process the event in nf pipelines, hence a new header